app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div id="exp-index" class="container-fluid col-sm-12 col-md-6 offset-md-3 col-lg-8 offset-lg-2 h-100">
+  <%= render partial: 'shared/navbar' %>
+  <%= render partial: 'shared/back_button' %>
+    <p class="mb-0 pb-0" style="text-align: center; font-family: proxima-nova, sans-serif;">
+      Expenses for:</p>
+    <h4 class="fw-bold mb-0 pb-0" style="text-align: center; font-family: proxima-nova, sans-serif;">
+      <%= @category.name %></h4>
+  <h4 class="fw-bold" style="text-align: center; font-family: proxima-nova, sans-serif; margin-bottom: 15px;">
+    Total Expenses: <%= number_to_currency(@category.total_expenses, precision: 2)%></h4>
+  <div class="container">
+    <table class="table">
+      <thead>
+        <tr class="bg-white">
+          <th>Name</th>
+          <th>Amount</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @expenses.each do |expense| %>
+          <tr class="bg-white my-1">
+            <td><%= expense.name %></td>
+            <td><%= number_to_currency(expense.amount, precision: 2) %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+
+    <% if can? :create, Expense %>
+      <%= link_to "Add Expense", new_user_category_expense_path(current_user, @category), class: "btn btn-primary",
+      style: "width: 100%; margin-top: 90px;" %>
+    <% end %>
```

Comment:
- [ ] Great work on adding the "Add Expense" button! However, there are still some changes that need to be made. According to the design, this button should be located at the bottom, which will take the user to the page to create a new transaction. Additionally, please follow the design guidelines and improve your user interface.

![image](https://github.com/jcmunav63/rails-budget-app/assets/55840999/c7fd0f58-3e79-42e3-abad-6b7cde626fd8)